docs/docs/guides/integrate/logout.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Logout
+---
+
+This guide shows you different concepts and use cases for logout and how to use it in ZITADEL.
+
+## OpenID Connect Single Logout 
+
+### Single Sign On (SSO) vs Single Logout (SLO)
+
+Single Sing On (SSO) allows a user to login once and be authenticated to several related applications.
+Single Logout (SLO) is the counterpart to this. With SLO a user can logout and terminate all related sessions, without actively logout from all systems.
+
+### OpenID Connect Logout
+
+The purpose of a logout is to terminate a user session. 
+Depending on how the session handling is implemented, there are different mechanisms that can be used.
+There are two different possibilities:
```

Comment:
@hifabienne would this not be better located in the section above instead of the OIDC Logout?